docs/websockets/websockets.html

```diff
@@ -0,0 +1,128 @@
+          <li class="nested-sidebar-item"><a href="../templating/static-file-server.html">Static File Server</a></li>
+          <li class="nested-sidebar-item"><a href="../templating/stencil.html">Stencil</a></li>
+          <li class="nested-sidebar-item"><a href="../templating/markdown.html">Markdown</a></li>
+        </ul>
+        <li class="sidebar-item collapsible">WebSockets</li>
+        <ul class="nested-sidebar-list content" style="max-height: 250px;">
+          <li class="nested-sidebar-item active"><a href="../websockets/websockets.html">What are WebSockets?</a></li>
+          <li class="nested-sidebar-item"><a href="../websockets/kiturawebsockets.html">Kitura-WebSocket Echo Server</a></li>
+        </ul>
+        <li class="sidebar-item collapsible">Deploying</li>
+        <ul class="nested-sidebar-list content">
+          <li class="nested-sidebar-item"><a href="../deploying/monitoring.html">Monitoring</a></li>
+          <li class="nested-sidebar-item"><a href="../deploying/ssl.html">Enabling SSL/TLS</a></li>
+          <li class="nested-sidebar-item"><a href="../deploying/docker.html">Docker</a></li>
+          <li class="nested-sidebar-item"><a href="../deploying/kubernetes.html">Kubernetes</a></li>
+          <li class="nested-sidebar-item"><a href="../deploying/cloud-foundry.html">Cloud Foundry</a></li>
+        </ul>
+        <li class="sidebar-item collapsible">Configuring</li>
+        <ul class="nested-sidebar-list content">
+          <li class="nested-sidebar-item"><a href="../configuring/enabling-nio.html">Enabling SwiftNIO</a></li>
+        </ul>
+      </ul>
+    </aside>
+    <div id="burgerIcon" class="burger-icon" onclick="showSidebar()">
+      <div class="burger-line"></div>
+      <div class="burger-line"></div>
+      <div class="burger-line"></div>
+    </div>
+    <div class="docs-item-2 search-container">
+    </div>
+    <nav class="docs-item-3 docs-nav">
+      <button id="api-button" class="apiref-button" type="button" name="button" onclick="window.open('https://ibm-swift.github.io/Kitura/')">API Reference</button>
+      <a class="nav-item" target="_blank" href="http://slack.kitura.io/">Need help?</a>
+      <a class="nav-item" target="_blank" href="https://github.com/IBM-Swift/kitura.io/issues">Found an issue?</a>
+    </nav>
+    <div id="doc-container" class="docs-item-4 docs-window">
+      <main>
+        <h1 class="heading-1">What are WebSockets?</h1>
+        <p class="block-text">In the early days of the web, a client and server relationship was viewed as following a <strong>single direction</strong> of workflow, the client would send a <strong>request</strong> to the server and the server would <strong>repsond</strong> to the client with the data requested.  However, as time progressed, this relationship evolved into one that became <strong>bidirectional</strong>, with both the client and server sending and receiving data to and from each other.</p>
```

Comment:
repsond?

Reply:
The <strong> in the text we only usually use for file names and package names so to look like our other guides that needs to be removed (unless it's for a package name).